crates/apollo-compiler/src/diagnostics.rs

```diff
@@ -425,3 +428,20 @@ pub struct UnsupportedLocation {
     #[help]
     pub help: Option<String>,
 }
+
+#[derive(Diagnostic, Debug, Error, Clone, Hash, PartialEq, Eq)]
+#[error("Subscription operations can not have an introspection field as a root field")]
```

Comment:
going to try to stick with Rust's error messaging conventions, and keep things lower cased 😅 

```suggestion
#[error("subscription operations can not have an introspection field as a root field")]
```